pkg/gardenlet/controller/seed/seed/reconciler_reconcile.go

```diff
@@ -221,6 +221,7 @@ func (r *Reconciler) checkMinimumK8SVersion(version string) (string, error) {
 
 const (
 	seedBootstrapChartName        = "seed-bootstrap"
+	kubeAPIServerPrefix           = "a-seed"
```

Comment:
```suggestion
	kubeAPIServerPrefix           = "api-seed"
```
WDYT, to make it consistent with `common.KubeAPIServerPrefix` used for shoots?